<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="rb_Hombre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>HOMBRE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_Mujer">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>MUJER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GENERO:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_Otro">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>OTRO</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border: 5px solid black;
	border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string>ESTADO CIVIL</string>
    </property>
    <widget class="QRadioButton" name="rb_Soltero">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	border: 5px solid black;
}</string>
     </property>
     <property name="text">
      <string>SOLTERO</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_Casado">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CASADO</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_UnionLibre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>UNIÃ“N LIBRE</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border: 5px solid black;
	border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string>GENERO</string>
    </property>
    <widget class="QRadioButton" name="rb_Mujer_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>MUJER</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_Hombre_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>HOMBRE</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_Otro_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>OTRO</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="rb_Hombre_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Casado</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_Otro_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Union Libre</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_Mujer_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Soltero</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Estado civil:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
